bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/driver/IOStream.java

```diff
@@ -39,6 +39,10 @@
     protected @Nullable OutputStream out = null;
     private boolean stopped = false;
```

Comment:
Because this is repeatedly checked in a while loop, it would be in your best interest to make this volatile.
```suggestion
    private volatile boolean stopped = false;
```

Reply:
done